.github/workflows/close-stale-feature-requests.yml

```diff
@@ -0,0 +1,44 @@
+name: Close stale feature requests
+on:
+  schedule:
+    # Run every day at 1:00 AM UTC.
+    - cron: 0 1 * * *
+
+env:
+  CLOSE_MESSAGE: >
+    There has been no activity on this feature request
+    and it is being closed. If you feel closing this issue is not the
+    right thing to do, please leave a comment.
+
+
+    For more information on how the project manages
+    feature requests, please consult the
+    [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).
+
+  WARN_MESSAGE: >
+    There has been no activity on this feature request for
+    5 months and it is unlikely to be implemented.
+    It will be closed 6 months after the last non-automated comment.
+
+    For more information on how the project manages
+    feature requests, please consult the
+    [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).
```

Comment:
```suggestion
# yamllint disable rule:empty-lines
env:
  CLOSE_MESSAGE: >
    There has been no activity on this feature request
    and it is being closed. If you feel closing this issue is not the
    right thing to do, please leave a comment.


    For more information on how the project manages
    feature requests, please consult the
    [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).

  WARN_MESSAGE: >
    There has been no activity on this feature request for
    5 months and it is unlikely to be implemented.
    It will be closed 6 months after the last non-automated comment.


    For more information on how the project manages
    feature requests, please consult the
    [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).
# yamllint enable
```
To satisfy the linter. (I have reported the error https://github.com/adrienverge/yamllint/issues/444)